app/src/main/java/com/it3048/mobiledeviceproject/dao/MeetingDatabase.kt

```diff
@@ -4,7 +4,7 @@ import androidx.room.Database
 import androidx.room.RoomDatabase
 import com.it3048.mobiledeviceproject.dto.Meeting
 
-@Database(entities = arrayOf(Meeting::class), version = 1)
+@Database(entities = [Meeting::class], version = 1)
```

Comment:
changed the array format to be in line with the current convention: https://kotlinlang.org/docs/whatsnew12.html#array-literals-in-annotations